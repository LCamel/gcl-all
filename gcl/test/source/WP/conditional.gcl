if False -> skip 
fi
{ True }