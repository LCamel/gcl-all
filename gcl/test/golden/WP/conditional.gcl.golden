### Result

[ Skip at ./test/source/WP/conditional.gcl [12-16] 1:13-17 #6c677a8bb491b68d
`True ∧ False
⇒
True`
explanation not available
, Conditional at ./test/source/WP/conditional.gcl [0-20] 1:1-2:3 #4ea8b990688e819d
`True
⇒
False`
explanation not available ]

### Source

if False -> skip 
fi
{ True }

### Trace

TraceLog "      [ save ] Parsed"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] TypeChecked"
TraceLog "      [ save ] Swept"
